src/app/edit-create-study-set/edit-create-study-set.component.ts

```diff
@@ -68,16 +68,22 @@ export class EditCreateStudySetComponent {
   loadStudySet() {
     getStudySetFromUrl(this.route, this.studySetService)
       .subscribe({
-        next: (sSet) => [
-          this.studySet = sSet,
-          this.isLoaded = true
-        ],
-        error: (e) => {if (e instanceof RouteParamNotFound) {
-          this.studySet.addCard();
-          this.isLoaded = true;
-        } else {
-          console.log(e);
-        }}
+        next: (sSet) => {
+          this.studySet = sSet;
+          if (sessionStorage.getItem('uid') === this.studySet.owner) {
+            this.isLoaded = true;
+          } else {
+            this.router.navigate(["home-page"]);
```

Comment:
Why did you decide to route to the homepage over the view page of the study set? Just a curiosity. Everything else looks good though.

Reply:
The reason was "I sleepy after riding the bus." I changed it to go to the view page. 

I also did more button re-styling.

The PR should be ready now.